New_PlatformGame/Assets/1. Script/GameManager.cs

```diff
@@ -21,6 +21,7 @@ public class GameManager : MonoBehaviour
     
```

Comment:
```suggestion
    private Vector3 CameraPositionTopRight; //카메라 위쪽의 Y좌표값을 가져오는 역할
    private Vector3 CameraPositionBottomLeft;
```
`변수 이름은 소문자로, 메서드와 클래스 이름은 대문자로 시작한다.` 라는 코드 컨벤션을 지키지 않았습니다.